use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+#Introduction
```

Comment:
Could you add a space after the headings hashes (here and below) as they aren't rendering correctly atm if you look at the preview.